src/overlays/actors/ovl_En_Holl/z_en_holl.h

```diff
@@ -4,6 +4,35 @@
 #include "ultra64.h"
 #include "global.h"
 
+#define ENHOLL_GET_TYPE(thisx) (((thisx)->params >> 6) & 7)
+#define ENHOLL_GET_SWITCH_FLAG(thisx) ((thisx)->params & 0x3F)
+
+/**
+ * Two kinds of holls:
+ *   Horizontal (H):
+ *     For the player to move horizontally between rooms.
+ *     Cuboid-shaped:
+ *      - height: 200 units high plus 50 units into the ground
+ *      - width: 400 units wide or, if narrow, 200 units wide
```

Comment:
I'm sure it'd be quick to lookup, but a cuboid has 3 dimensions. Which dimension is missing? also, is width (left <-> right) or depth? (refence is player walking into cuboid)

Reply:
Depth is missing, (depth = along the dimension the player traverses the cuboid (maybe I'm not using height/width/depth properly here though))
as well as height for the vertical (cylinder) holls

The thing is there isn't a single depth/height, the distances vary by type, and some types have several "depth distance triggers"
For example one type has a depth range in which the room on the other side loads, and another inner range in which the visible plane fades away, what would be its depth

I'm not sure how to document all of it in a few words